src/_guides/language/sound-dart.md

```diff
@@ -11,20 +11,27 @@ This guide tells you why and how to write sound (type safe) Dart code.
 You'll learn how to use strong mode to enable soundness, as well as
 how to substitute types safely when overriding methods.
 
+_**Strong mode**_ is a sound static
+type system that uses a combination of static and runtime checks to
+ensure your code is type safe&mdash;that you can never see a value
+whose runtime type does not match its static type.
+With strong mode enabled (in an implementation that has both the
+static and runtime checks), Dart is a sound language.
+
 <aside class="alert alert-info" markdown="1">
-  The terms **sound** Dart, **strong mode** Dart, and **type safe** Dart
-  are sometimes used interchangeably. _Strong mode_ is a sound static
-  type system that uses a combination of static and runtime checks to
-  ensure your code is type safe&mdash;that you can never see a value
-  whose runtime type does not match its static type.
-  With strong mode enabled (in an implementation that has both the
-  static and runtime checks), Dart is a sound language.
+#### Terminology
```

Comment:
This format is unusual. It's kind of like the summaries we have in some tutorials (e.g. https://www.dartlang.org/tutorials/language/streams), but the background is different _and_ this isn't a summary.

The usual way of formatting a note (which I think should work here) is like this:

**Terminology note:** The terms.... **Classic** Dart....